zendframework/zendframework/ZF2014-05.yaml

```diff
@@ -2,12 +2,6 @@ title:     Anonymous authentication in ldap_bind() function of PHP, using null b
 link:      http://framework.zend.com/security/advisory/ZF2014-05
 cve:       ~
 branches:
-    2.0.x:
-        time:     2014-09-16 22:00:00
-        versions: [>=2.0.0,<2.0.99]
-    2.1.x:
-        time:     2014-09-16 22:00:00
-        versions: [>=2.1.0,<2.1.99]
```

Comment:
Looking at the security advisory, Zend Framework before `1.12.9` was also affected. Though I'm not sure if that version will be covered in the repository at all.